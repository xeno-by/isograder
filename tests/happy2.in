* Test [/15]
** Sub1 [/7]
-1: blah
** Sub2 [/8]
-2: foo
*** Sub21 [/4]
+1: bar
*** Sub22 [/4]
-1: baz