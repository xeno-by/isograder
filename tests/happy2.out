EXIT CODE:
0

STDOUT:
* Test [11/15]
** Sub1 [6/7]
-1: blah
** Sub2 [5/8]
-2: foo
*** Sub21 [4/4]
+1: bar
*** Sub22 [3/4]
-1: baz

STDERR:

